python/res/enkf/hook_manager.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, workflow_list, config_content=None, config_dict=None):
 
             # RUNPATH_FILE #
             runpath_file_name = config_dict.get(ConfigKeys.RUNPATH_FILE, ConfigKeys.RUNPATH_LIST_FILE)
-            runpath_file_path = os.path.join(config_dir, runpath_file_name)
+            runpath_file_path = os.path.normpath(os.path.join(config_dir, runpath_file_name))
```

Comment:
why not realpath?

Reply:
it compares to abspath. Also when joining two paths might produce `.../...`, which fails in tests.